### DISCUSSION TITLE
don't import

### DISCUSSION THREAD

**Discussion by shu-a-be** [lnccbrown/HSSM#276] [Q&A] [answered]
Hi,
I am using a Mac with an M1 chip. When I install hssm and try to import it, I get the following error message:
"runtimeerror: this version of jaxlib was built using avx instructions, which your cpu and/or operating system do not support. you may be able work around this issue by building jaxlib from source."

If anyone knows how to resolve this issue, please let me know.

Thank you.

**Comment by digicosmos86** [comment, marked as answer]
Hi,

This is because somehow a version of `jaxlib` that was built for other platforms was installed. Here's a few thing that you can do to resolve this:

1. Install everything from your system terminal. Before installing anything, use the `arch` command and ensure that the output is `arm64`. If it's not, it means that the terminal is running on Rosetta. Turn off rosetta before moving forward.
2. Install `hssm` into a fresh virtual environment. Sometimes other packages in the environment might also cause the wrong version of `jaxlib` to be installed.
3. Ensure that your Python interpreter is built for arm64. You can check that by getting into the Python REPL and check the version and the build.

**Reply by shu-a-be** [reply]
Hi,
Thank you for the advice.
I'll try it out right away.
